code/modules/mod/mod_activation.dm

```diff
@@ -161,8 +166,10 @@
 		seal_part(helmet, seal = !active)
 	if(do_after(wearer, activation_step_time, wearer, MOD_ACTIVATION_STEP_FLAGS))
 		to_chat(wearer, span_notice("Systems [active ? "shut down. Parts unsealed. Goodbye" : "started up. Parts sealed. Welcome"], [wearer]."))
-		if(ai)
-			to_chat(ai, span_notice("<b>SYSTEMS [active ? "DEACTIVATED. GOODBYE" : "ACTIVATED. WELCOME"]: \"[ai]\"</b>"))
+		// SKYRAT EDIT START - pAIs in MODsuits
+		if(mod_pai)
```

Comment:
Could probably be a trinary operator single line change

Reply:
It could, but only if I was leaving support for AIs, which I won't.